--- c-ares-1.22.0/src/lib/ares_sysconfig_files.c.orig	2023-11-14 13:14:37.000000000 +0100
+++ c-ares-1.22.0/src/lib/ares_sysconfig_files.c	2023-11-14 23:04:57.557257498 +0100
@@ -528,6 +528,7 @@
     switch (error) {
       case ENOENT:
       case ESRCH:
+      case EACCES:
         break;
       default:
         DEBUGF(fprintf(stderr, "fopen() failed with error: %d %s\n", error,
@@ -556,6 +557,7 @@
     switch (error) {
       case ENOENT:
       case ESRCH:
+      case EACCES:
         break;
       default:
         DEBUGF(fprintf(stderr, "fopen() failed with error: %d %s\n", error,
@@ -586,6 +588,7 @@
     switch (error) {
       case ENOENT:
       case ESRCH:
+      case EACCES:
         break;
       default:
         DEBUGF(fprintf(stderr, "fopen() failed with error: %d %s\n", error,
@@ -616,6 +619,7 @@
     switch (error) {
       case ENOENT:
       case ESRCH:
+      case EACCES:
         break;
       default:
         DEBUGF(fprintf(stderr, "fopen() failed with error: %d %s\n", error,
